src/test/java/xyz/niflheim/stockfish/util/StringUtil.java

```diff
@@ -2,8 +2,6 @@
 
 public class StringUtil {
     public static final String START_REGEX = "^";
-    public static final String GREETING_STOCKFISH =
```

Comment:
Why was this line removed?

Reply:
Then it is better to write developers, substituting the Stockfish version using regex, as you indicated in the tests

Reply:
1. This was removed as the welcome message has changed following the next version after 10.

2. Developers are no longer in the welcome message as (at least) of version 16 i.e.: "Stockfish 16.1 by the Stockfish developers (see AUTHORS file)". The common denominator for all welcome messages remains the word "Stockfish" followed by a space and then a 2 digit version number e.g.: "Stockfish 10", "Stockfish 16" etc.

Reply:
OK